app/django/meridien/issues/views.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+from django.http.response import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework import status
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
+from rest_framework.permissions import IsAuthenticated
+
+from issues.models import Issue
+from issues.serializers import IssueSerializer
+
+import sys
+sys.path.append('../')
+from meridien import views_template
+
+# Create your views here.
+@api_view(['GET', 'POST', 'DELETE'])
+@permission_classes([IsAuthenticated])
+@csrf_exempt
+def issue_list(request):
+    response = views_template.obj_list(request, Issue, IssueSerializer)
+    return response
+
+@api_view(['GET', 'PUT', 'DELETE'])
+@permission_classes([IsAuthenticated])
```

Comment:
same here... but this one u need to protect so that ppl can't anyhow edit the reviews